backend/app/api/v1/endpoints/custom_sites.py

```
@@ -0,0 +1,312 @@
+        log.articles_found = len(articles_data)
+        saved_count = 0
+        
+        # 記事を保存
+        news_fetcher = NewsFetcher(db)
+        for article_data in articles_data:
+            try:
+                news_fetcher._save_article(article_data)
+                saved_count += 1
+            except Exception as e:
+                print(f"Error saving article: {str(e)}")
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)